snomed/com.b2international.snowowl.snomed.reasoner.server/src/com/b2international/snowowl/snomed/reasoner/server/classification/EquivalentConceptMerger.java

```diff
@@ -110,7 +110,7 @@ public void fixEquivalencies() {
 		final SnomedDeletionPlan deletionPlan = new SnomedDeletionPlan();
 		for (final Concept conceptToKeep : equivalentConcepts.keySet()) {
 			final Collection<Concept> conceptsToRemove = equivalentConcepts.get(conceptToKeep);
-			switchToEquivalentConcept(conceptToKeep, conceptsToRemove, inboundRelationshipMap);
+			switchToEquivalentConcept(conceptToKeep, conceptsToRemove, inboundRelationshipMap, equivalentConcepts);
```

Comment:
Do we need the extra `Multimap` argument for this check? You already have concept to keep, which will be the same source/destination for all outbound/inbound relationships, and we only need to compare the destination/source against the value for this concept in the `Multimap`, ie. `conceptsToRemove`.

Reply:
After discussion, the answer is yes.